src/components/service-metrics/service-metrics.ts

```diff
@@ -8,10 +8,41 @@ import { IContext } from '../app';
 import { CLOUD_CONTROLLER_ADMIN, CLOUD_CONTROLLER_GLOBAL_AUDITOR, CLOUD_CONTROLLER_READ_ONLY_ADMIN } from '../auth';
 import { fromOrg, IBreadcrumb } from '../breadcrumbs';
 import { drawMultipleLineGraphs } from '../charts/line-graph';
+import { UserFriendlyError } from '../errors';
 import elasticacheServiceMetricsTemplate from './elasticache-service-metrics.njk';
 import rdsServiceMetricsTemplate from './rds-service-metrics.njk';
 import unsupportedServiceMetricsTemplate from './unsupported-service-metrics.njk';
 
+interface IRange {
+  readonly period: moment.Duration;
+  readonly rangeStart: moment.Moment;
+  readonly rangeStop: moment.Moment;
+}
+
+export async function resolveServiceMetrics(ctx: IContext, params: IParameters): Promise<IResponse> {
+  const rangeStop = moment();
+  const timeRanges: {[key: string]: moment.Moment} = {
+    '1h': rangeStop.clone().subtract(1, 'hour'),
+    '3h': rangeStop.clone().subtract(3, 'hours'),
+    '12h': rangeStop.clone().subtract(12, 'hours'),
+    '24h': rangeStop.clone().subtract(24, 'hours'),
+    '7d': rangeStop.clone().subtract(7, 'days'),
+    '30d': rangeStop.clone().subtract(30, 'days'),
+  };
+  const rangeStart = timeRanges[params.offset] || timeRanges['1h'];
```

Comment:
I think the default should be `24h`, rather than `1h` (at least, that's what it was before).